docs/intro/quickstart/workflow.rst

```diff
@@ -0,0 +1,111 @@
+.. _ref_quickstart_workflow:
+
+===============================
+A Smoother Development Workflow
+===============================
+
+Part of building an application involves making changes to the schema, so let's take some time to make a comfortable development workflow for ourselves. Spending a bit of time now will make it easier to stay in the flow of development.
+
+Staying in sync
+===============
+
+.. edb:split-section::
+
+  When your schema changes, we need to regenerate the query builder files to pick up the new changes. This slows us down a bit, so the next workflow improvement is to add a hook script so that any time the schema changes, the query builder files are regenerated automatically after the migration is applied.
+
+  .. code-block:: toml-diff
+    :caption: gel.toml
+
+      [gel]
+      server-version = 6.0
+    +
+    + [hooks]
+    + schema.update.after = "npx @gel/generate edgeql-js"
+
+.. edb:split-section::
+
+  Let's make some changes to our schema and update our code. Our first change will be to add a property to our ``Deck`` type that stores the link to all of the cards in the deck ordered by the ``order`` property on the ``Card`` type. We will create a computed property, and use a back link from the ``Card`` type to the ``Deck`` type.
+
+  .. code-block:: sdl-diff
+    :caption: dbschema/default.gel
+
+      module default {
+        type Deck {
+          required name: str;
+          description: str;
+    +
+    +     cards := (select .<deck[is Card] order by .order);
```

Comment:
I think we should set max line length to 50 or less. Or just prefer to spend more space vertically than horizontally. People on mobile will say thank you. But even on desktop:


![CleanShot 2025-02-07 at 17 15 16@2x](https://github.com/user-attachments/assets/5ac6ed8c-1941-49a9-94cd-222e82bbe2c9)

Reply:
@jaclarke we need to make horizontal scrollbars always visible on macos for code snippets

Reply:
Related: I also think we should use more of the horizontal space of the page for the code snippets. Screenshots and code feel cramped due to the space we leave for the in-page TOC.